index.css

```diff
@@ -61,12 +61,99 @@
     color: #5da8ef;
 }
 
+/* section primary-slider */
+
+.primary-slider {
+    position: relative;
+    padding-top: 43px;
+}
+
+.primary-slider__content {
+     /*linear-gradient(to right, rgba(255, 255, 255, 0.2) 0%, rgba(255, 255, 255, 0) 100%);*/
+     position: absolute;
+     transform: translateY(-50%);
+     top: 50%;
+}
+
+.primary-slider__subtitle {
+    opacity: 0.9;
+    color: #5da8ef;
+    font-size: 9px;
+    font-weight: 700;
+    line-height: 66px;
+    text-transform: uppercase;
+    letter-spacing: 6px;
+}
+
+.primary-slider__title {
+    margin: 0;
+    color: #0b0c11;
+    font-family: "Playfair Display";
+    font-size: 62px;
+    font-weight: 700;
+    line-height: 64px;
+}
+
+.primary-slider__paragraph {
+    width: 47%;
+    color: #8f8f8f;
+    font-size: 14px;
+    font-weight: 300;
+    line-height: 24px;
+}
+
+.primary-slider__image {
+    box-shadow: -7px 19px 20px 5px darkgrey;
+}
+
+
+.slider-controls {
+    position: absolute;
+    top: 93%;
```

Comment:
Are there any, more convenient properties you can use to align it at the bottom?